src/renderer/components/opds/Entry.tsx

```diff
@@ -41,9 +41,11 @@ class Entry extends React.Component<EntryProps, undefined> {
                         to={ route }
                     >
                         <span className={styles.flux_title}>{entry.title}</span>
+                        {entry.subtitle ? <span className={styles.flux_subtitle}>{entry.subtitle}</span> : <></>}
                         {
                             (entry.publicationCount) ?
                                 (
+                                    // FIXME livres in french
```

Comment:
Let's just remove "livres" altogether. No need for a label here, just a number. IMO :)

Reply:
fixed